google/cloud/bigquery/v2/minimal/internal/job_client_test.cc

```diff
@@ -31,30 +32,30 @@ using ::google::cloud::testing_util::StatusIs;
 using ::testing::HasSubstr;
 using ::testing::Return;
 
+Job GetJob() {
```

Comment:
optional: I might call this `MakeTestJob()`. Especially if you plan to reuse it for a different API that creates a `Job` for testing. Up to you.